sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamBigQuerySqlDialect.java

```diff
@@ -239,6 +254,18 @@ private void unparseTrim(SqlWriter writer, SqlCall call, int leftPrec, int right
     writer.endFunCall(trimFrame);
   }
 
+  private void unparseDoublePositiveINFWrapperFunction(SqlWriter writer) {
+    writer.literal("CAST('+inf' AS FLOAT64)");
```

Comment:
Maybe comment here: "There is no direct ZetaSQL literal representation of NaN or infinity"?

Reply:
Comment Added.